### PR TITLE
Fixed godmode

### PR DESCRIPTION
**Pull request by mgejay**
I think that was supposed to be true.

### CODE REVIEW

**Comment by Wild1145** [comment]
While that also has fixed it, the reason its not working is because of line 31 :)

**Comment by Wild1145** [comment]
@Scuph Any updates with this?

**Comment by mgejay** [comment]
@Wild1145 You want me to remove line 31?

**Comment by Wild1145** [comment]
@Scuph Nope, just read it and you should see very quickly exactly what the mistake is.

**Comment by mgejay** [comment]
@Wild1145 All updates I thought were necessary have been done, please tell me if you wish for anything else to be done on this command.

File: src/com/superiornetworks/icarus/commands/Command_god.java [hunk 1]
```diff
@@ -28,11 +28,10 @@ public boolean run(CommandSender sender, Command cmd, String label, String[] arg
                 return true;
             }
             Player player = (Player) sender;
-            if (!ICM_SqlHandler.hasDoomHammer(player.getName()))
```
Comment (Wild1145) [inline src/com/superiornetworks/icarus/commands/Command_god.java:31]: You have removed this (Great) but not replaced it with what should be there, a check for if they have god mode.

**Comment by Wild1145** [comment]
@Scuph You have made most of the changes, because you changed the way the command is loaded it will also need to be removed from the plugin.yml.

I am going to ask that you re-do these small changes and open a new PR to keep the number of commits down and it clean.

**Pull request closed without merging**